ibis/backends/snowflake/__init__.py

```diff
@@ -395,27 +395,38 @@ def _make_batch_iter(
                 for t in cur.cursor.fetch_arrow_batches()
             )
 
+    @contextlib.contextmanager
+    def _use(self, ident):
```

Comment:
Can you give this a clearer name (`_use_schema`?)

Reply:
Yep, will rename to `_use_schema`